docs/editor/integrated-terminal.md

```diff
@@ -283,7 +283,7 @@ Look at the setting details to see the complete list of default commands.
 
 By default, when a chord keybinding is the highest priority keybinding, it will always skip the terminal shell (bypassing `terminal.integrated.commandsToSkipShell`) and be evaluated by VS Code instead of the terminal. This is typically the desired behavior unless you're on Windows/Linux and want your shell to use ctrl+k (for bash, this cuts the line after the cursor). This can be disabled with the following setting:
 
-```json
+```jsonc
```

Comment:
There are not comments in this code block so `json` for the language type is fine.